consensus/ethash/consensus.go

```diff
@@ -587,28 +587,43 @@ var (
 // AccumulateRewards credits the coinbase of the given block with the mining
 // reward. The total reward consists of the static block reward and rewards for
 // included uncles. The coinbase of each uncle block is also rewarded.
-func accumulateRewards(config ctypes.ChainConfigurator, state *state.StateDB, header *types.Header, uncles []*types.Header) {
+// func AccumulateRewards(config ctypes.ChainConfigurator, state *state.StateDB, header *types.Header, uncles []*types.Header) {
```

Comment:
@meowsbits @iquidus can you review this function and the ones that being used inside for any logical issues?

Reply:
Have done. LGTM, and the passing tests indicate positive too.